README.md

```diff
@@ -8,16 +8,16 @@ Cross chain events are very important in the future of crypto. Exchanging tokens
 
 ## Overview
 
-We have a smart contract bridge on each network, the bridge on one chain will receive and lock the tokens, then it will emmit an event that will be served to the bridge on the other chain. There is a Federation in charge of sending the event from one contract to the other.
+We have a bridge smart contract on each network, the bridge on one chain will receive and lock the ERC20 tokens, then it will emit an event that will be served to the bridge on the other chain. There is a Federation in charge of sending the event from one contract to the other. Once the bridge on the other chain receives the event from the Federation, it mints the tokens on the mirror ERC20 contract.
 See the [FAQ](./docs/FAQ.md) to know more about how it works!
 
+<p align="center">
+  <img src="./docs/images/token-bridge-diagram.png"/>
+</p>
```

Comment:
```suggestion
```

Reply:
the Html tag is to actually display the image, if you add it as [Bridge Diagram](./docs/images/token-bridge-diagram.png) it just adds it as an hiper link.
So im going to leave it as it is